common/src/main/java/com/microsoft/identity/common/internal/util/ClockSkewManager.java

```diff
@@ -23,71 +23,88 @@
 package com.microsoft.identity.common.internal.util;
 
 import android.content.Context;
+import android.content.SharedPreferences;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.VisibleForTesting;
 
 import com.microsoft.identity.common.internal.cache.ISharedPreferencesFileManager;
 import com.microsoft.identity.common.internal.cache.SharedPreferencesFileManager;
+import com.microsoft.identity.common.java.interfaces.IKeyPairStorage;
 
 import java.util.Calendar;
 import java.util.Date;
 
-public class ClockSkewManager implements IClockSkewManager {
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+/**
+ * Deprecated
+ * Currently serving as an adapter of {@link com.microsoft.identity.common.java.util.ClockSkewManager}
+ */
+@SuppressFBWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS",
+        justification = "This class kept its original name to avoid breaking change during the refactoring process." +
+                "Once the process is done, this class will be removed entirely. ")
+public class ClockSkewManager extends com.microsoft.identity.common.java.util.ClockSkewManager {
 
     private static final class PreferencesMetadata {
 
         private static final String SKEW_PREFERENCES_FILENAME =
                 "com.microsoft.identity.client.clock_correction";
-
-        private static final String KEY_SKEW = "skew";
     }
 
-    private ISharedPreferencesFileManager mClockSkewPreferences;
-
     @VisibleForTesting(otherwise = VisibleForTesting.NONE)
     public ClockSkewManager(@NonNull final ISharedPreferencesFileManager manager) {
-        mClockSkewPreferences = manager;
+        super(new IKeyPairStorage<Long>() {
```

Comment:
At this point, I've seen so many of these very similar looking inline implementations of `IKeyPairStorage`...can't we just define it once in its own class and just initialize here and other places where it is needed?